CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## master / unreleased
 
+* [CHANGE] Add `json` struct tags to overrides' `Limits` struct in addition to `yaml` tags.
```

Comment:
Can you add the PR number and link as well?

Reply:
Done :)